polkadot/node/service/chain-specs/paseo.json

```diff
@@ -4,9 +4,20 @@
   "chainType": "Live",
   "bootNodes": [
     "/dns/paseo.bootnode.amforc.com/tcp/30333/wss/p2p/12D3KooWFD81HC9memUwuGMLvhDDEfmXjn6jC4n7zyNs3vToXapS",
-    "/dns/paseo.bootnode.amforc.com/tcp/30344/p2p/12D3KooWFD81HC9memUwuGMLvhDDEfmXjn6jC4n7zyNs3vToXapS"
+    "/dns/paseo.bootnode.amforc.com/tcp/30344/p2p/12D3KooWFD81HC9memUwuGMLvhDDEfmXjn6jC4n7zyNs3vToXapS",
+    "/dns/paseo-bootnode.radiumblock.com/tcp/30333/p2p/12D3KooWADeaC8zag4Qrb4GosSn65MmfVZztRPMaBdgZnQqXRo",
+    "/dns/paseo-bootnode.radiumblock.com/tcp/30335/wss/p2p/12D3KooWeayZC8zag4Qrb4GosSn65MmfVZztRPMaBdgZnQqXRo",
```

Comment:
@al3mart  [same fix](https://github.com/paseo-network/runtimes/commit/6b2adf9f557e2041e94a7c6aeeaa0dde344a03a1) required here or going to get [parse errors](https://github.com/paseo-network/runtimes/issues/18) using --chain paseo.

Reply:
Thanks @hitchhooker! I have pushed the fix